src/phrases/phrases-nn.ts

```diff
@@ -740,6 +740,7 @@ const phrases = {
       descriptionlanguage: 'Beskrivelse-språk',
       url: 'Lenke',
       date: 'Dato',
+      sides: 'Hvilken side skal bilde vises',
```

Comment:
Nynorsk versjon av det jeg skrev over

Reply:
"På kva for ei side biletet skal visast"